app/views/transactions/_activity_row.html.erb

```diff
@@ -1,17 +1,18 @@
-<% activity_id = transaction.dig('activities', '_id') %>
-<% application = transaction.dig('magi_medicaid_application') ? transaction.dig('magi_medicaid_application') : application_from_activity(transaction.dig('correlation_id'))%>
-<tr id="row_<%= activity_id %>">
-  <td><%= transaction.dig('application_id') %></td>
-  <td><%= transaction.dig('primary_hbx_id') %></td>
-  <td><%= fpl_year(application) %></td>
-  <td><%= transaction.dig('activities', 'event_key')&.humanize&.upcase %></td>
-  <td><%= transaction.dig('activities', 'status') %></td>
-  <td><%= link_to transaction.dig('activities', 'updated_at'), transaction_path(transaction['_id'], anchor: "activty_#{activity_id}") %></td>
+<% ar = activity_row %>
```

Comment:
we flip twice this name, why we don't name it as needed from the loop on index.html.erb, however usually in rails the full name is preferred, what I'm trying to say is that this is probably better used as activity_row instead of ar

Reply:
oh yeah, good call!